torch/_refs/__init__.py

```diff
@@ -503,6 +505,84 @@ def _isnan(a: TensorLikeType) -> TensorLikeType:
 )
 
 
+# TODO replace with finfo and iinfo prims
+def _minimum_limit(dtype: torch.dtype):
```

Comment:
Should `_minimum_limit` and `_maximum_limit` be implemented with `finfo` and `iinfo` prims?

Reply:
I replaced `_minimum_limit` and `_maximum_limit` with for `minimum_value` and `maximum_value` prims.